test/e2e/README.md

```diff
@@ -192,6 +192,49 @@ You can load the new image into the kind cluster using the command below:
 kind load docker-image projects.registry.vmware.com/antrea/antrea-ubuntu:latest --name <kind_cluster_name>
 ```
 
+## Running the e2e tests on existing K8s cluster
+
+If you already have a K8s cluster on which Antrea has been pushed, the following steps should be done for running the e2e tests.
+
+First, you should create the ssh config file and add the ssh information of each Node in the cluster to it. You can add the ssh information like the example below:
+
+```bash
+Host Control-Plane-Node
```

Comment:
Since this is placeholder like Control-Plane-IP, please use `< >` to indicate it.